dist/ember-charts.js

```diff
@@ -1584,12 +1584,19 @@ define('ember-charts/components/pie-chart', ['exports', 'module', 'ember', './ch
     numSlices: _Ember['default'].computed.alias('finishedData.length'),
 
     // Offset slices so that the largest slice finishes at 12 o'clock
+    // If there is an excessively largest slice, have all of the smaller slices
+    // concentrate on the left hand side.
     startOffset: _Ember['default'].computed('finishedData', function () {
       var data = this.get('finishedData');
       var sum = data.reduce(function (p, d) {
         return d.percent + p;
       }, 0);
-      return _.last(data).percent / sum * 2 * Math.PI;
+
+      if (_.last(data).percent / sum > 0.65) {
+        return 4 / 3 * Math.PI;
```

Comment:
Why do we hardcode for this special case? If the largest slice is 90% of the pie, wouldn't we want the `startOffset` to show that rather than only offsetting 66% (4/3 * Pi) ?